packages/application/src/index.ts

```diff
@@ -396,11 +495,13 @@ export class Application<T extends Widget> {
     let startups = Private.collectStartupPlugins(this._pluginMap, options);
 
     // Generate the activation promises.
-    let promises = startups.map(id => {
-      return this.activatePlugin(id).catch(error => {
+    let promises = startups.map(async id => {
```

Comment:
The original version here seems more straightforward because it doesn't wrap the `activate` promise in an additional `await`.